client/src/components/achievements/Achievement.js

```diff
@@ -123,4 +124,9 @@ const Achievement = ({ achievement }) => {
   );
 };
 
+// props validation
```

Comment:
curious: is there a reason why we're leaving line comments above some of the `PropTypes` implementations and not on others? We could either leave comments on top of all of them or remove them entirely to maintain consistency.

Reply:
The most possible reason is that I forgot to add on others. Will add them now XD